lesson13/script.js

```diff
@@ -0,0 +1,83 @@
+function showLoadingImg(){
+    const div = document.createElement('div');
+    const loadingImg = document.createElement('img');
+    div.id = "js-loading";
+    loadingImg.src = "img/loading-circle.gif";
+    loadingImg.alt = "ローディング画像";
+    document.body.insertAdjacentElement('afterbegin',div).appendChild(loadingImg);
+}
+
+function removeLoadingImg(){
+    document.getElementById('js-loading').remove();
+}
+
+async function fetchdata(){
+    const endpoint = "https://api.json-generator.com/templates/szdgGQcOLXuk/data?access_token=hu4bc7qh9znx2m8f53mn4mz2hryvdntkavwbw8j0";
+    const response = await fetch(endpoint);
+    //responseがokではないとき
+    if(!response.ok){
+    throw new Error(`${response.status}:${response.text}`);
+    }
+    //responseがokのとき
```

Comment:
IMO
I felt this comment out was unnecessary.
This is because the meaning is immediately obvious when you look at the code.

Reply:
Thank you for your comment!

Looking back later, I thought I didn't understand, so I added a comment.
This is because I didn't fully understand the meaning of the code.

As you say, it's easy for anyone who understands the code to understand, so I'll remove the comment.

Reply:
I fixed https://github.com/nakagawamai/js-lesson/pull/15/commits/480363fe2a2017267dd73cf2b3d73ef13cb2a092